Library/Homebrew/cmd/commands.rb

```diff
@@ -25,10 +25,10 @@ def self.call
       # TODO: Put this check_invalid_options() method such that it doesn't have to be here
       check_invalid_options
 
-      if argv.include? "--quiet"
+      if quiet_present_in_argv?
```

Comment:
I think this is a step in the right direction, although it's a bit of a mouthful in its current state. I think `quiet?` or even `opt_quiet?` would be preferable, if you're worried about someone clobbering existing methods by adding an option.